docs/src/pages/components/progress/progress.md

```diff
@@ -114,3 +114,8 @@ If you need to perform 30 re-renders per second or more, we recommend disabling
   transition: none;
 }
 ```
+
+### IE 11
+
+The circular progress component animation on IE 11 is degraded. The stroke dash animation is not working (equivalent to `disableShrink`) and the circular animation wobbles.
+You can solve the latter issue following [#5735](https://github.com/mui-org/material-ui/issues/5735).
```

Comment:
Can we give concrete advice how to do it? Reading the whole issue and guessing the correct solution isn't all that helpful.

Reply:
Why would we want to do that? The related issue is 4 years old and had received little traction in that period.

Reply:
> We are targeting advanced users here, right?

Why would we do that.

> Why would we want to do it?

I don't know how to answer that. Same reason why you would document anything.

Reply:
I assume that IE 11 support is only good for legacy, that only users with resources will go the extra mile to apply the change. I also assume that our time it limited, when we spend it on figuring out how to help this audience, we don't spend it helping another.